src/integrationTest/java/com/mongodb/kafka/connect/MongoSinkConnectorTest.java

```diff
@@ -41,78 +41,59 @@
 class MongoSinkConnectorTest extends MongoKafkaTestCase {
 
     @Test
-    @DisplayName("Ensure simple producer sends data")
-    void testASimpleProducerSmokeTest() {
```

Comment:
This test was testing the producer worked and not the connector.  Its not really needed so I removed it.